app/serializers/workflow_serializer.rb

```diff
@@ -49,4 +49,8 @@ def content
   def retirement
     @model.retirement_with_defaults
   end
+
+  def finished_at
+    @model.finished_at || ""
```

Comment:
something like that. we agreed a while back to always serialize a default value (mostly via db defaults) so i just added it in for consistent behaviour.